styles/components/_buttons.scss

```diff
@@ -0,0 +1,72 @@
+.button {
+  // Positioning
+  position: relative;
+
+  // Display & Box Model
+  border-radius: $border-radius;
+  border: 0;
+  padding: 1rem;
+  box-shadow: 0px 4px 4px rgba(0, 0, 0, 0.25);
+  min-width: 400px;
+
+  // Color
+  background-color: $color-primary;
+  color: $text-color-primary;
+
+  // Text
+  font-weight: 500;
+  font-size: $font-size-primary;
+  line-height: 1;
```

Comment:
Are you sure of one? Typical `lh` is set to about 160%. Check some articles about golden ratio typography

Reply:
Yes, indeed it is a mistake, figma says "line-height: 44px;". Needs to be corrected.